src/main/kotlin/racing/service/Racing.kt

```diff
@@ -0,0 +1,24 @@
+package racing.service
+
+import racing.model.Car
+
+class Racing(carCount: Int) {
+    private var cars: List<Car> = List(carCount) { Car() }
+
+    fun play() {
+        cars.forEach { car ->
+            val moving = car.run()
+            print(START_POINT_SYMBOL)
+            repeat(moving) {
+                print(CAR_SYMBOL)
+            }
+            println()
```

Comment:
print() 함수로 문자 하나하나를 출력하는 것이 아니라, String의 `repeat()` 확장함수를 활용해서 출력할 위치 값을 완성해서 한번에 출력하도록 변경해보면 어떨까요? 😃
```suggestion
            val position = CAR_SYMBOL.repeat(moving)
            println(position)
```